src/NServiceBus.Gateway/Routing/Sites/ConfigurationBasedSiteRouter.cs

```diff
@@ -4,6 +4,13 @@ namespace NServiceBus.Gateway.Routing.Sites
 
     class ConfigurationBasedSiteRouter : IRouteMessagesToSites
     {
+        readonly IDictionary<string, Site> sites;
```

Comment:
Can you guys update the resharper code styles and apply a format on this code. The field should be lower down. Readonly is not necessary,